### PR TITLE
Switch com.valvesoftware.Steam beta to freedesktop-sdk 22.08beta

### PR DESCRIPTION
**Pull request by nanonyme**

### CODE REVIEW

File: com.valvesoftware.Steam.yml [hunk 1]
```diff
@@ -107,16 +107,17 @@ add-extensions:
...
     autodelete: false
 
-  org.freedesktop.Platform.ffmpeg_full.i386:
-    directory: lib32/ffmpeg
-    add-ld-path: .
-    version: '21.08'
-    no-autodownload: true
-    autodelete: false
+#  TODO: freedesktop-sdk 22.08 doesn't currently have this
+#  org.freedesktop.Platform.ffmpeg_full.i386:
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:115]: We don't need to remove extension point for a non-existent extension. It'll just stay empty, and should be installed and enabled when/if it becomes available.

Comment (nanonyme) [reply com.valvesoftware.Steam.yml:115]: Ah, cool.

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
22.08beta glibc is broken. Trying to fix it through https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/merge_requests/8987. Let's revisit this later.

**Comment by gasinvein** [comment]
bot, build

**Comment by 1player** [comment]
Trying out the new beta, upon startup it complains about:

* org.freedesktop.Platform.Compat.i386//22.08beta
* org.freedesktop.Platform.GL32.default//22.08beta

not being installed and fails with a missing libc error. Shouldn't these be installed automatically/as dependencies?

Installing them manually fixes it.

**Comment by nanonyme** [comment]
This is common issue with beta app.

**Comment by gasinvein** [comment]
> fails with a missing libc error

Ugh, this shouldn't be happening after #879 anymore. It should've shown a GUI error popup instead. I guess

**Comment by 1player** [comment]
It did show a GUI error, but then proceeded to load and broke.

**Comment by nanonyme** [comment]
See https://github.com/flathub/com.valvesoftware.Steam/wiki/Using-beta-branch-of-this-app for installation instructions

**Pull request merged**